src/essential_validator.py

```diff
@@ -95,7 +95,7 @@ def essentialValidate(configs, job_queue, mongo_dao):
                             mongo_dao.set_submission_validation_status(validator.submission, None, submission_meta_status, None)
                     else:
                         log.error(f'Invalid message: {data}!')
-
+                    log.info(f'Processed the job!')
```

Comment:
Ideally, we can change the log like "Processed the job; metadata/file/essential"